test/e2e/scenarios/pod_selector.go

```diff
@@ -2,24 +2,106 @@ package scenarios
 
 import (
 	. "github.com/onsi/ginkgo"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	framework "github.com/submariner-io/submariner/test/e2e/framework"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+var f = framework.NewDefaultFramework("podselectornp")
```

Comment:
We normally put the framework inside the describe part, I think it was for cleanup reasons, can't remember now.

Reply:
I did this way so that the same framework can be used for control plane tests as well.